MAUI/ListView/emptyview.md

```diff
@@ -0,0 +1,320 @@
+             xmlns:local="clr-namespace:EmptyViewDemo">
+  <ContentPage.Resources>
+    <ResourceDictionary>
+      <ContentView x:Key="SingleView">
+        <Label Text="No Items" FontSize="18" FontFamily="Roboto-Regular" 
+               HorizontalTextAlignment="Center" VerticalOptions="CenterAndExpand"/>
+      </ContentView>
+      <ContentView x:Key="MultiView">
+        <StackLayout VerticalOptions="CenterAndExpand">
+          <Label Text="&#xe725;" FontSize="40"
+                 FontFamily="{OnPlatform iOS=ListViewFontIcons, MacCatalyst=ListViewFontIcons, Android=ListViewFontIcons.ttf#, UWP=ListViewFontIcons.ttf#ListViewFontIcons}"
+                 HorizontalTextAlignment="Center" />
+          <Label TextColor="#666666" Text="No Items" FontSize="16" FontFamily="Roboto-Regular" HorizontalTextAlignment="Center" />
+        </StackLayout>
+      </ContentView>
+    </ResourceDictionary>
+  </ContentPage.Resources>
+
+  <ContentPage.Content>
+    <Grid>
+      <Grid.RowDefinitions>
+        <RowDefinition Height="30"/>
+        <RowDefinition Height="30"/>
+        <RowDefinition Height="*" />
+      </Grid.RowDefinitions>
+      <SearchBar x:Name="filterText" 
+                 FontSize="16"                              
+                 Placeholder="Filter Inventory" TextChanged="SearchBar_TextChanged"/> 
+      <CheckBox Grid.Row="1" x:Name="checkBox" IsChecked="False" 
+                             CheckedChanged="CheckBox_CheckedChanged"/>                                                  
+      <syncfusion:SfListView Grid.Row="2" x:Name="listView"
+                             ItemsSource="{Binding Items}"
+                             ItemSize="56"
+                             EmptyView="{StaticResource SingleView}">                   
+      </syncfusion:SfListView>
+    </Grid>
+  </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+listView.EmptyView = Resources["SingleView"];
+
+checkBox.CheckedChanged += CheckBox_CheckedChanged;
+{% endhighlight %}
+{% endtabs %}
+
+`SfListView.EmptyView` is changed based on value of CheckBox.IsChecked property at run time.
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void CheckBox_CheckedChanged(object sender, CheckedChangedEventArgs e)
+{
+  if(e.Value)
+    listView.EmptyView = Resources["MultiView"];
+  else
+    listView.EmptyView = Resources["SingleView"];
+}
+{% endhighlight %}
+{% endtabs %}
+
+![SingleView EmptyView in .NET MAUI ListView](Images/emptyview/maui-listview-singleview-emptyview.jpg)
+
+![MultiView EmptyView in .NET MAUI ListView](Images/emptyview/maui-listview-multiview-emptyview.jpg)
+
+Download the entire source code from GitHub [here](https://github.com/SyncfusionExamples/how-to-change-empty-view-at-run-time-in-.net-maui-listview).
+
+## Change an EmptyViewTemplate at runtime
+
+The `EmptyView` appearance can be changed at run time based on its value using `EmptyViewTemplate` property.

```

Comment:
The `EmptyView` appearance can be changed at run time based on its value using the `EmptyViewTemplate` property.